### PR TITLE
Identify CYF-only content

### PR DESCRIPTION
**Pull request by Dedekind561**
## What does this change?

- Label and identify all content that is only relevant to the CYF organisation

### Common Content?

Block/s

### Common Theme?

Issue number: #issue-number

### Org Content?

Module | Sprint | Page Type | Block Type

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

**Comment by Dedekind561** [comment]
@FridIsar Thanks for labelling this content here. Is there anything else you want to add to this piece of work?

**Comment by SallyMcGrath** [comment]
@FridIsar BOOP

File: .github/ISSUE_TEMPLATE/module-rewrite.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ assignees: ""
 Module(s):
 Week(s):
 
-Rewriting a week in the curriculum will often involve updating different sections of the curriculum, including the backlog and workshops. You can use this issue to organise the different changes in the curriculum content for a week at CYF.
+Rewriting a week in the curriculum will often involve updating different sections of the curriculum, including the backlog and workshops. You can use this issue to organise the different changes in the curriculum content for a week at <!-- CYF-ONLY -->CYF<!-- END-CYF-ONLY -->.
```
Comment (FridIsar) [inline .github/ISSUE_TEMPLATE/module-rewrite.md:13]: All these comments should be removed right? As it makes sense to keep CYF when it comes to contributions

File: common-content/en/module/databases/integration-with-node/index.md [hunk 2]
```diff
@@ -75,9 +75,9 @@ app.get("/customers", function (req, res) {
 In the code above:
 
 - We first import the `Pool` class from the pg library, which is used to connect to a database
-- We create a new connection (`db`) where we specify the credentials to connect to the cyf_hotel database
+- We create a new connection (`db`) where we specify the credentials to connect to the {{<our-name>}}\_hotel database
```
Comment (FridIsar) [inline common-content/en/module/databases/integration-with-node/index.md:78]: extra "\"

File: common-content/en/module/databases/integration-with-node/index.md [hunk 3]
```diff
@@ -75,9 +75,9 @@ app.get("/customers", function (req, res) {
...
 - We then create a new `/customers` endpoint where we use the method `query()` to send a SQL query to load all the customers from the table `customers` and return the results with `result.rows`. The `query` method returns a Promise: so we can access the returned rows using a `.then` block. You can write any valid SQL query that you learned in the `query()` method!
 
 **_Note:_** There is no semicolon to end the SQL statement.
 
-Start your server with `node server.js` and try to reach the `/customers` endpoint to see the list of customers currently available in your `cyf_hotels` database. You can try to create/update/delete customers using psql to verify that your API always returns what is stored in your database.
+Start your server with `node server.js` and try to reach the `/customers` endpoint to see the list of customers currently available in your `{{<our-name>}}_hotels` database. You can try to create/update/delete customers using psql to verify that your API always returns what is stored in your database.
```
Comment (FridIsar) [inline common-content/en/module/databases/integration-with-node/index.md:83]: Extra "s"

**Review by FridIsar — APPROVED** [review]
Love the use of the shortcode! 
- If you agree I can now remove all comments, especially as when it comes to contributing the word CYF should be kept. 

- There might be some issues with e.g. mcb-hotels-api if the code forked by the trainee has cyf?

File: .github/ISSUE_TEMPLATE/module-rewrite.md (hunk 1 above)
Comment (SallyMcGrath) [reply .github/ISSUE_TEMPLATE/module-rewrite.md:13]: I don't think there are any remaining in the common content, only in our own org content.

**Pull request merged**